src/main/java/com/playmonumenta/networkrelay/BroadcastCommand.java

```diff
@@ -93,6 +131,18 @@ public void networkRelayMessageEvent(NetworkRelayMessageEvent event) {
 			return;
 		}
 
+		@Nullable JsonPrimitive serverTypeJson = data.getAsJsonPrimitive("server_type");
+		if (serverTypeJson != null) {
+			@Nullable String serverTypeString = serverTypeJson.getAsString();
+			if (serverTypeString != null) {
+				NetworkRelayAPI.ServerType commandType;
+				commandType = NetworkRelayAPI.ServerType.fromString(serverTypeString);
+				if (!ACCEPTED_SERVER_TYPES.contains(commandType)) {
+					return;
+				}
+			}
+		}
```

Comment:
I think these Nullable uses are fine, but I think we should resist the urge to annotate them everywhere. The nullable checker doesn't actually care what you annotate variables within methods - only the interfaces matter (class variables, parameters). Everything else is determined by the return value of the function. 

Seems fine to manually annotate things, but I think it doesn't actually help reduce NPEs because what you annotate a variable as might not actually be what is returned (or the return value nullability could even change later!)